spec/lib/sidekiq/throttled/patches/super_fetch_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require "sidekiq/throttled/patches/super_fetch"
+begin
+  require "sidekiq/pro/super_fetch"
+  SIDEKIQ_PRO_AVAILABLE = true
+rescue LoadError
+  # Sidekiq Pro is not available
+  SIDEKIQ_PRO_AVAILABLE = false
+end
+
+RSpec.describe Sidekiq::Throttled::Patches::SuperFetch do
+  if SIDEKIQ_PRO_AVAILABLE
```

Comment:
As we're already loading sidekiq-pro in the patch - we can shorten this to:

```suggestion

RSpec.describe Sidekiq::Throttled::Patches::SuperFetch do
  if defined? Sidekiq::Pro::SuperFetch
```

Reply:
Really appreciate all the feedback! 💪🏽